.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 node_modules/
 .sass-cache
 .jekyll-metadata
+.bundle/
```

Comment:
Additonally Git indicates that "This branch is 4 commits ahead, 200 commits behind master. "  which is the start of the commit history for the repo.